src/main/java/org/folio/inventory/consortium/handlers/MarcInstanceSharingHandlerImpl.java

```diff
@@ -26,29 +42,32 @@
 public class MarcInstanceSharingHandlerImpl implements InstanceSharingHandler {
 
   private static final Logger LOGGER = LogManager.getLogger(MarcInstanceSharingHandlerImpl.class);
-
+  private static final String CONSORTIUM_PREFIX = "CONSORTIUM-";
   private final RestDataImportHelper restDataImportHelper;
-
   private final InstanceOperationsHelper instanceOperations;
-
+  private final EntitiesLinksService entitiesLinksService;
+  private final Storage storage;
   private final Vertx vertx;
 
-  public MarcInstanceSharingHandlerImpl(InstanceOperationsHelper instanceOperations, Vertx vertx) {
+  public MarcInstanceSharingHandlerImpl(InstanceOperationsHelper instanceOperations, Storage storage, Vertx vertx) {
     this.vertx = vertx;
     this.instanceOperations = instanceOperations;
     this.restDataImportHelper = new RestDataImportHelper(vertx);
+    this.entitiesLinksService = new EntitiesLinksServiceImpl(vertx, vertx.createHttpClient());
```

Comment:
I am not too knowledgeable about the vertx http client. Should there be only one client for the app? should a client recreated somewhere else be used?

Reply:
Moved initialization of HTTP client to consumer, so we have one client, thanks